api.bs

```diff
@@ -598,9 +598,13 @@ dictionary PrivateAttributionConversionOptions {
   sequence<DOMString> intermediarySites = [];
 };
 
+dictionary PrivateAttributionConversionResult {
+  required Uint8Array report;
+};
+
 [SecureContext, Exposed=Window]
 partial interface PrivateAttribution {
-  [Throws] Promise<Uint8Array> measureConversion(PrivateAttributionConversionOptions options);
+  [Throws] Promise<PrivateAttributionConversionResult> measureConversion(PrivateAttributionConversionOptions options);
 };
 </xmp>
```

Comment:
This probably needs more.  We don't really describe the return value very well when describing the function, so maybe it is worth doing that down on line 666(old)/670(new) below.

Reply:
I agree, but we're already missing a lot of detail here. For example, `Encrypt the report` and `Return the encrypted report` do not explain how the `Uint8Array` is actually produced, so it might make more sense to fill that in later.